### PR TITLE
Docs: Update whats-new 9.3 with auth related news

### PR DESCRIPTION
**Pull request by gamab**
**What is this feature?**

Update Whats-new 9.3 with auth related news:
- LDAP - Role mapping improvements
- Azure AD OAuth2 - New option to always fetch groups from the Graph API
- RBAC - List token's permissions

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/whatsnew/whats-new-in-v9-3.md [hunk 1]
```diff
@@ -110,6 +110,35 @@ $ grafana-cli user-manager conflicts validate-file <filepath>
...
+and make user roles editable from within Grafana.
+
+## Azure AD OAuth2 - New option to always fetch groups from the Graph API
+
+If you use Azure AD OAuth2 authentication and use `SecurityEnabled` groups that you don't want Azure to embed in the
+authentication token, you can force Grafana to use Microsoft's Graph API instead.
+
+Use the [`force_use_graph_api` configuration option]({{< relref
+"../setup-grafana/configure-security/configure-authentication/azuread/#force-fetching-groups-from-microsoft-graph-api" >}})
+when configuring Azure AD authentication to force Grafana to fetch
```
Comment (IevaVasiljeva) [inline docs/sources/whatsnew/whats-new-in-v9-3.md:131]: Looks like the sentence is missing an ending.
```suggestion
when configuring Azure AD authentication to force Grafana to fetch groups using Graph API.
```

File: docs/sources/whatsnew/whats-new-in-v9-3.md [hunk 2]
```diff
@@ -110,6 +110,35 @@ $ grafana-cli user-manager conflicts validate-file <filepath>
...
+
+## RBAC - List token's permissions
+
+We added a new endpoint to help users diagnose permissions-related issues with user and token authorization.
+[This endpoint]({{< relref "../developers/http_api/access_control/#list-your-permissions" >}}) allows users to get the
+full list of RBAC permissions associated with their token.
+
+For more details, see the related service accounts [section]({{< relref
```
Comment (IevaVasiljeva) [inline docs/sources/whatsnew/whats-new-in-v9-3.md:139]:
```suggestion
For more details, see the related service accounts [documentation]({{< relref
```

**Review by IevaVasiljeva — APPROVED** [review]
Left some nits and one sentence is missing an ending, but other than that this is a great summary of auth improvements!

File: docs/sources/whatsnew/whats-new-in-v9-3.md (hunk 1 above)
Comment (gamab) [reply docs/sources/whatsnew/whats-new-in-v9-3.md]: Seems like something disappeared when I shuffled things around :sweat_smile:

**Pull request merged**